serverManager/common/source/SessionServerApp.cpp

```diff
@@ -35,14 +35,33 @@
 
 namespace
 {
-constexpr char sessionManagementPrefix[]{"/tmp/rialto-"};
 constexpr int maxPlaybackSessions{2};
 constexpr int maxWebAudioPlayers{1};
+const std::string sessionManagementSocketDefaultDir{"/tmp"};
+const std::string sessionManagementSocketDefaultName{"/rialto-"};
 
 std::string generateSessionManagementSocket()
```

Comment:
maybe change name of this function to generateSessionManagementSocketName or generateSessionManagementSocketPath (I actually like Path more), since it doesn't generate socket?

Reply:
changed.